hw03_karimh.py

```diff
@@ -6,22 +6,122 @@
 # Purpose: To create a code using functions, and submit a pull request.
 # Google Doc Link: https://docs.google.com/document/d/12MLXaEkr6jyzp5jVL2UXoEJTj2VyV12LwCJIk-eW6fs/edit?usp=sharing
 
+
 import turtle
 
-def function_1():
-    "include docstring"
+# Assigning name to the turtle
+Heer = turtle.Turtle()
+
+
+def tyre_1(rad, th, H):
+    "rad = radius of the circle"
+    "th = thickness of the tyre"
+    "Makes the left tyre of the car"
+
+    # Gives black color to the tyre
+    H.fillcolor("black")
+    H.begin_fill()
+
+    # Attributes of the tyre
+    H.pensize(th)
+    H.circle(rad)
+    H.end_fill()
+
+
+def tyre_2(rad, th, H):
+    "rad = radius of the circle"
+    "th = thickness of the tyre"
+
+    H.fillcolor("black")
+    H.begin_fill()
+
+    H.pensize(th)
+    H.circle(rad)
+    H.end_fill()
+
+
+def lower_body(l, w, H):
+    "l = length of the lower rectangular body of car"
+    "w = width of the lower rectangular body of car"
+    "Makes the lower body of the car which looks like a big rectangle"
+
+    # Colors the lower body blue
+    H.fillcolor("blue")
+    H.begin_fill()
+
+    # Gives movement directions to the turtle
+    H.left(180)
+    H.forward(l)
+    H.right(90)
+    H.forward(w)
+    H.right(90)
+    H.forward(l)
+    H.right(90)
+    H.forward(w)
+    H.end_fill()
+
 
+def upper_body(ul,uw, H):
+    "ul = length of the upper rectangular body"
+    "uw = width of the upper rectangular body"
 
-def function_2():
-    "docstring"
+    H.fillcolor("grey")
+    H.begin_fill()
 
-def function_3():
-    "docstring"
+    H.left(180)
+    H.forward(ul)
+    H.right(90)
+    H.forward(uw)
+    H.right(90)
+    H.forward(ul)
+    H.right(90)
+    H.forward(uw)
+    H.end_fill()
 
-def fucntion_4():
-    "docstring"
 
 def main ():
-    #call functions
+    # Creates method for the screen
+    wn = turtle.Screen()
+    wn.bgpic("chitral-valley.gif")     # Adds a background picture
+
+    wn.colormode(255)
+    wn.bgcolor("lightblue")
+    Heer.speed(0)
```

Comment:
`Heer = turtle.Turtle()` should go here

Reply:
Moved Heer = turtle.Turtle() before Heer.speed(0))